llvm/test/lit.site.cfg.py.in

```diff
@@ -55,6 +55,7 @@ config.linked_bye_extension = @LLVM_BYE_LINK_INTO_TOOLS@
 config.linked_exampleirtransforms_extension = @LLVM_EXAMPLEIRTRANSFORMS_LINK_INTO_TOOLS@
 config.have_tf_aot = @LLVM_HAVE_TF_AOT@
 config.have_tflite = @LLVM_HAVE_TFLITE@
+config.enable_profcheck = @LLVM_ENABLE_PROFCHECK@
```

Comment:
It doesn't look like you're using this anywhere?

Reply:
yes, but I have it there in case we need to apply to tests to disable them when this is enabled. It's fairly canonical, so figured I'd add it here.